### DISCUSSION TITLE
Number of small and large pores on Mobi ?

### DISCUSSION THREAD

**Discussion by Salih-Benamara** [Open-Systems-Pharmacology/Forum#1548] [Questions & Problems]
Hi all 😃

Please, I would like to know if there is a paramter or a value for the number of small and large pores on Mobi (PK-Sim) ? If not, can we configure this setting (Number of pores) by changing a spesific parameter ? Thank You 😄

**Comment by StephanSchaller** [comment]
HI Salih,

I believe this is not an explicit parameter, but would be calculated by Por Surface Area divided by pore size

**Reply by Salih-Benamara** [reply]
I see, thank you very much for the reply !

**Comment by Christoph27** [comment]
Hi Salih,
as Stephan wrote the number of pores is not an explicit parameter. Also, the total pore surface area is not an explicit parameter, but it occurs conceptionally to estimate the permeabilities for small and large pores. Direct parameters used in the model related to this are the pore radii of small & large pores, the hydraulic conductivity of the endothelium, the fraction of flow vial large and small pores (and the viscosity). You can look at equations 10 and 13 in https://doi.org/10.1007/s10928-017-9559-4 for more details.

**Reply by Salih-Benamara** [reply]
Hello Christoph,

Yes, I see the mechanism in these equations. Thank you very much for the reply :) !
